docs/docs/getting-started/install.md

```diff
@@ -263,6 +263,11 @@ To [create a Constellation cluster](../workflows/create.md), you need the follow
 * `compute.firewalls.delete`
 * `compute.firewalls.get`
 * `compute.firewalls.update`
+* `compute.forwardingRules.create`
+* `compute.forwardingRules.delete`
+* `compute.forwardingRules.get`
+* `compute.forwardingRules.setLabels`
```

Comment:
I don't think we need these. Or do I miss something?

Reply:
we set them in the iam repo. but not sure if we strictly need them. maybe @3u13r knows more